src/main/java/no/nav/fo/veilarbregistrering/kafka/KafkaConfig.java

```diff
@@ -59,6 +66,42 @@ Properties kafkaProperties() {
         return properties;
     }
 
+    @Bean
+    KontaktBrukerOpprettetKafkaConsumer kontaktBrukerOpprettetKafkaConsumer(
+            KafkaConsumer kafkaConsumer,
+            UnleashService unleashService,
+            DatakvalitetOppholdstillatelseService datakvalitetOppholdstillatelseService
+    ) {
+        return new KontaktBrukerOpprettetKafkaConsumer(
+                kafkaConsumer,
+                unleashService,
+                datakvalitetOppholdstillatelseService);
+    }
+
+    @Bean
+    KafkaConsumer kafkaConsumer() {
+        KafkaConsumer kafkaConsumer = new KafkaConsumer(kafkaConsumerProperties());
+        kafkaConsumer.subscribe(Collections.singletonList("aapen-arbeid-arbeidssoker-kontaktbruker-opprettet" + getEnvSuffix()));
+        return kafkaConsumer;
+    }
+
+    @Bean
+    Properties kafkaConsumerProperties() {
+        Properties properties = new Properties();
+        properties.put(ConsumerConfig.BOOTSTRAP_SERVERS_CONFIG, getenv("KAFKA_SERVERS"));
+        properties.put(ConsumerConfig.GROUP_ID_CONFIG, getGroupId());
+        properties.put(KafkaAvroDeserializerConfig.SCHEMA_REGISTRY_URL_CONFIG, getenv("KAFKA_SCHEMA"));
+        properties.put(ConsumerConfig.KEY_DESERIALIZER_CLASS_CONFIG, StringDeserializer.class);
+        properties.put(ConsumerConfig.VALUE_DESERIALIZER_CLASS_CONFIG, KafkaAvroDeserializer.class);
+        properties.put(KafkaAvroDeserializerConfig.SPECIFIC_AVRO_READER_CONFIG, true);
+        properties.put(ConsumerConfig.AUTO_OFFSET_RESET_CONFIG, "earliest");
```

Comment:
@chriswk - Ser denne konfigurasjonen greit ut? Må/bør vi også inkludere properties for sikkerhet her, som vi har gjort for producer?

Reply:
Ja, dere må ha sikkerhetsoppsett for konsumering også, ellers vil du ikke få tilgang til topicet.